runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
Adding to consul or to some other discovery tool, register your container internally etc. Apart from this I have real use-case in docker where I need event when I can say client that application is ready. Do you really have plans to start `sidecar` processes?